cmd/blockchaincmd/add_validator.go

```diff
@@ -53,21 +55,17 @@ var (
 	pop                                 string
 	remainingBalanceOwnerAddr           string
 	disableOwnerAddr                    string
-	rpcURL                              string
-	aggregatorLogLevel                  string
-	aggregatorLogToStdout               bool
 	delegationFee                       uint16
 	errNoSubnetID                       = errors.New("failed to find the subnet ID for this subnet, has it been deployed/created on this network?")
 	errMutuallyExclusiveDurationOptions = errors.New("--use-default-duration/--use-default-validator-params and --staking-period are mutually exclusive")
 	errMutuallyExclusiveStartOptions    = errors.New("--use-default-start-time/--use-default-validator-params and --start-time are mutually exclusive")
 	errMutuallyExclusiveWeightOptions   = errors.New("--use-default-validator-params and --weight are mutually exclusive")
 	ErrNotPermissionedSubnet            = errors.New("subnet is not permissioned")
-	aggregatorExtraEndpoints            []string
-	aggregatorAllowPrivatePeers         bool
 	clusterNameFlagValue                string
 	createLocalValidator                bool
 	externalValidatorManagerOwner       bool
 	validatorManagerOwner               string
+	validatorManagerFlags               flags.ValidatorManagerFlags
```

Comment:
lets stop using the same variable in different commands. we can take the opportunity here
to define a struct per command, with all his flags, and defined a struct instance for the command,
and use it

Reply:
combined into flags instead, no more using local variable in different cmd files. This will allow for better testing of values